draft-ietf-quic-http.md

```diff
@@ -324,7 +325,7 @@ transport parameter `initial_max_uni_streams`.
 
 If the stream header indicates a stream type which is not supported by the
 recipient, the remainder of the stream cannot be consumed as the semantics are
-unknown. Recipients of unknown stream types MAY trigger a QUIC STOP_SENDING
+unknown. Recipients of unknown stream types MAY send a QUIC STOP_SENDING
```

Comment:
The use of "trigger" is deliberate, since an HTTP implementation cannot directly send transport frames.  It can ask the transport implementation to do so.